.circleci/config.yml

```diff
@@ -406,15 +406,6 @@ jobs:
           name: Run Tests
           command: yarn --cwd=packages/contractkit test
 
-      - run:
```

Comment:
Why remove these?

Reply:
This came out of talking to Mariano. This test fails when `contracktkit` depends on unpublished packages, namely `@celo/utils` and so we removed this check under because it will always fail for any PR with breaking changes to `@celo/utils`. Do you think that's fair?

Reply:
Oh and it would prevent us from moving to the scheme we are suggesting of pointing to an unpublished version number, so if we want to keep this check we will need to abandon that scheme.

Reply:
yes. We can really install the commited version of the package, since their dependencies are also not published... this type of test don't work anymore for us